docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,53 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+Upgrading Alluxio in Kubernetes is easy. You can just change the Alluxio docker image version to your desired upper version.
```

Comment:
- don't say that it is easy, it can  be frustrating for users if they are not able to do it right
- don't use "upper version" as that terminology isn't defined anywhere
Change this to 
Step 1: Upgrade the docker image version tag to the version you wish to upgrade to
Step 2: Restart the ... pod (specify the order to restart master / worker and commands to use)
...
Step n: Verify ...